src/main/scala/is/hail/utils/richUtils/RichHadoopConfiguration.scala

```diff
@@ -29,7 +29,7 @@ class RichHadoopConfiguration(val hConf: hadoop.conf.Configuration) extends AnyV
       os
   }
 
-  private def open(filename: String): InputStream = {
+  def open(filename: String): InputStream = {
```

Comment:
We had these private because they are unsafe (don't clean up resources).  Better to use `readFile` or `readDataFile` (or add another variant if none of those work for you).